pymc/sampling/jax.py

```diff
@@ -144,14 +144,16 @@ def get_jaxified_graph(
     return jax_funcify(fgraph)
 
 
-def get_jaxified_logp(model: Model, negative_logp=True) -> Callable:
+def get_jaxified_logp(model: Model, negative_logp=True) -> Callable[[PointType], jnp.ndarray]:
     model_logp = model.logp()
     if not negative_logp:
         model_logp = -model_logp
     logp_fn = get_jaxified_graph(inputs=model.value_vars, outputs=[model_logp])
+    names = [v.name for v in model.value_vars]
 
-    def logp_fn_wrap(x):
-        return logp_fn(*x)[0]
+    def logp_fn_wrap(x: PointType) -> jnp.ndarray:
+        p = [x[n] for n in names]
+        return logp_fn(*p)[0]
```

Comment:
This is a breaking change. What about adding an optional kwarg for `point_fn` with the default value being False?

Reply:
I see, will do that

Reply:
I better do that in get_jaxified_graph then

Reply:
I was thinking that... not sure either way. Go with your gut feeling

Reply:
One reason not to do it in get_jaxified_graph is that not all graphs just have named or uniquely named variables, so maybe let's do it just for the logp

Reply:
done

Reply:
Did you push?

Reply:
no:<, I forgot, pushed now